query.py

```diff
@@ -0,0 +1,394 @@
+import json
+import requests
+import networkx as nx
+
+
+class Query:
+    def __init__(
+        self,
+        service_base_url: str,
+        CapabilityStatement_path: str = None,
+    ) -> None:
+        self.base = service_base_url
+        self.possible_references = self._get_rev_include_possibilitie(
+            CapabilityStatement_path
+        )
+        """Instantiate the class and create the query object
```

Comment:
Maybe explain quickly what the `Query` is there for, I don't think this line learns us a lot :)

Reply:
You're totaly right. I'll make a draft after we clear up the variable naming :)